docs/UserGuide.md

```diff
@@ -482,12 +625,12 @@ Action | Format, Examples
 
 Action | Format, Examples
 -------|------------------------------
-**Add** | `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG] ...` <br> e.g., `c-add n/James Ho p/22224444 e/81234567 a/Blk 123 ABC Road t/Friday t/PartTime`
-**List** | `c-list`
+**Add** | `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG]` <br> e.g., `c-add n/James Ho p/22224444 e/81234567 a/Blk 123 ABC Road t/Friday t/PartTime`
+**List** | `c-active-list`, `c-archive-list`
```

Comment:
To be more consistent, instead of "`c-active-list`, `c-archive-list`", maybe can change to the code below:

Suggestion:

1. **List all active(unarchived) employees:**  `c-active-list`<br>**List all archived employees:**  `c-archive-list`<be>